.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 inventory/vagrant_ansible_inventory
 inventory/group_vars/fake_hosts.yml
+inventory/host_vars/k8s-0*/
```

Comment:
You can just write it as inventory/host_vars. There's no need to exclude just a certain host pattern.